asset/quickstart/ExportAssets/ExportAssets.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright(c) 2018 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START export_assets_quickstart]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Asset.V1Beta1;
+using System;
+
+
+public class ExportAssets
+{
+    public static void Main(string[] args)
+    {
+        string projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");

```

Comment:
Use a literal string here, and then replace it in runTests.ps1, like is done here: https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/master/trace/api/runTests.ps1

Reply:
I tried to use literal string for bucketName, and replace it with Environment.GetEnvironmentVariable by using runTests.ps1, somehow it complains that "ExportAssets.cs(26,66): error CS1002: ; expected", I am wondering if you know what I missed there? I guess the ps1 didn't replace it correctly.